app/src/main/java/com/scouts/kitchenplaner/ui/view/recipeforproject/RecipeForProject.kt

```diff
@@ -16,17 +16,192 @@
-        Text("Recipe $recipeID is shown", color = Color.Red)
-        Text(text = "available Links to other sides are: ")
+fun RecipeForProjectScreen(
+    project: Project,
+    mealSlot: MealSlot,
+    recipeID: Long,
+    onNavigateToAlternative: (Long) -> Unit,
+    onNavigateToRecipeDetails: (Long) -> Unit,
+    recipeViewModel: RecipeForCookingViewModel = hiltViewModel()
+) {
+    var recipeInitialized by remember { mutableStateOf(false) }
+
+    LaunchedEffect(key1 = null) {
+        recipeViewModel.getRecipe(project, mealSlot, recipeID)
+        recipeInitialized = true
     }
 
-}
\ No newline at end of file
+    if (recipeInitialized) {
+        val recipeForCooking by recipeViewModel.recipeForCooking.collectAsState()
+
+        Column (modifier = Modifier
+            .fillMaxWidth()
+            .verticalScroll(rememberScrollState())
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(50.dp)
+                    .padding(10.dp),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = recipeForCooking.name,
+                    fontSize = 8.em,
+                    fontWeight = FontWeight.Black,
+                    color = MaterialTheme.colorScheme.onSecondaryContainer,
+                    overflow = TextOverflow.Ellipsis
+                )
+
+                IconButton(onClick = { onNavigateToRecipeDetails(recipeID) }) {
+                    Icon(
+                        imageVector = Icons.Filled.Edit,
+                        contentDescription = "Edit Recipe",
+                        tint = MaterialTheme.colorScheme.onSecondaryContainer
+                    )
+                }
+            }
+
+            HorizontalDivider()
+
+            Text(
+                text = "Anzahl Personen: ${recipeForCooking.people}",
+                modifier = Modifier
+                    .padding(top = 10.dp)
+                    .fillMaxWidth(),
+                textAlign = TextAlign.Center
+            )
+
+            IngredientsInput(
+                ingredientGroups = recipeForCooking.ingredientGroups,
+                editable = false,
+                modifier = Modifier.padding(10.dp)
+            )
+
+            InstructionInput(
+                instructions = recipeForCooking.instructions,
+                editable = false,
+                modifier = Modifier.padding(10.dp)
+            )
+
+            DisplayAlternatives(
+                alternatives = recipeForCooking.alternatives,
+                modifier = Modifier.padding(10.dp),
+                onNavigateToAlternative = onNavigateToAlternative
+            )
+        }
+    } else {
+        Text("Waiting for the recipe to be loaded")
+    }
+}
+
+@Composable
+fun DisplayAlternatives(
```

Comment:
It would be nice, if the recipe itself is not displayed at the alternative recipe when it is indeed the alternative recipe.
(There should be a link to the main recipe instead, and this recipe should be marked as the main recipe)

Edit: I think we already talked about this issue, but I'm not sure what our conclusion was. (Fixing this in another issue, because there is a lot happening in other layers.. ?

Reply:
We decided to fix this once we could assign allergen people to recipes as then we need to redesign this area anyway.